tests/org.eclipse.swt.tests.win32/JUnit Tests/org/eclipse/swt/tests/win32/AllWin32Tests.java

```diff
@@ -23,7 +22,7 @@
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
 	Test_org_eclipse_swt_dnd_DND.class,
-	Test_org_eclipse_swt_widgets_Display.class,
+//	Test_org_eclipse_swt_widgets_Display.class,
```

Comment:
Just wondering as it is named `AllWin32Tests` should it not actually *compile* when compiling for/with windows?